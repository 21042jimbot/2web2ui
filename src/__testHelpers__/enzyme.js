export const findButton = (wrapper, buttonLabel) =>
  wrapper.findWhere(
    (node) => node.type() === 'button' && new RegExp(buttonLabel).test(node.text())
  );
