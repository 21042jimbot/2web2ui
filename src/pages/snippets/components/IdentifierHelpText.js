import React from 'react';
import { ExternalLink } from 'src/components/links';
import { LINKS } from 'src/constants';

const IdentifierHelpText = () => (
  <React.Fragment>
    Use this unique identifier to{' '}
    {
      <ExternalLink to={LINKS.SNIPPET_SUBSTITUTION_REFERENCE}>
        reference your snippet in a template
      </ExternalLink>
    }
  </React.Fragment>
);

export default IdentifierHelpText;
