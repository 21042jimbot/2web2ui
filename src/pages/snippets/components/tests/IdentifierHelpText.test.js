import React from 'react';
import { shallow } from 'enzyme';
import IdentifierHelpText from '../IdentifierHelpText';

describe('IdentifierHelpText', () => {
  it('renders message with link', () => {
    const wrapper = shallow(<IdentifierHelpText />);
    expect(wrapper).toMatchSnapshot();
  });
});
