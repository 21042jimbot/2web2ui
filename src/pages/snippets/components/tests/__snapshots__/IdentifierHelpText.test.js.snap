// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`IdentifierHelpText renders message with link 1`] = `
<Fragment>
  Use this unique identifier to
   
  <ExternalLink
    to="https://developers.sparkpost.com/api/substitutions-reference/#header-snippets"
  >
    reference your snippet in a template
  </ExternalLink>
</Fragment>
`;
